### ISSUE TITLE
Parser fails to extract the right information in some cases

### ISSUE THREAD

**Issue by delbarital** [secynic/ipwhois#204]
lets take for example the whois raw response for  **66.69.255.255**:

```
NetRange:       66.68.0.0 - 66.69.255.255
CIDR:           66.68.0.0/15
NetName:        RR-SOUTHWEST-2BLK
NetHandle:      NET-66-68-0-0-1
Parent:         NET66 (NET-66-0-0-0-0)
NetType:        Direct Allocation
OriginAS:
Organization:   Time Warner Cable Internet LLC (RRSW)
RegDate:        2001-01-25
Updated:        2011-07-06
Comment:        ADDRESSES WITHIN THIS BLOCK ARE NON-PORTABLE
Ref:            https://whois.arin.net/rest/net/NET-66-68-0-0-1



OrgName:        Time Warner Cable Internet LLC
OrgId:          RRSW
Address:        13820 Sunrise Valley Drive
City:           Herndon
StateProv:      VA
PostalCode:     20171
Country:        US
RegDate:
Updated:        2017-01-30
Comment:        Allocations for this OrgID serve Road Runner residential customers out of the Austin, TX and Tampa Bay, FL RDCs.
Ref:            https://whois.arin.net/rest/org/RRSW

ReferralServer:  rwhois://ipmt.rr.com:4321

OrgTechHandle: IPTEC-ARIN
OrgTechName:   IP Tech
OrgTechPhone:  +1-703-345-3416
OrgTechEmail:  abuse@rr.com
OrgTechRef:    https://whois.arin.net/rest/poc/IPTEC-ARIN

OrgAbuseHandle: ABUSE10-ARIN
OrgAbuseName:   Abuse
OrgAbusePhone:  +1-703-345-3416
OrgAbuseEmail:  abuse@rr.com
OrgAbuseRef:    https://whois.arin.net/rest/poc/ABUSE10-ARIN
```

We clearly see that the country is **US** and the organization name should be **IP Tech**, or at least the ISP name **Time Warner Cable Internet LLC**. However, this is the output:

```
{'cidr': '66.68.0.0/15',
 'country': None,
 'end_address': '66.69.255.255',
 'events': [{'action': u'last changed',
             'actor': None,
             'timestamp': u'2011-07-06T17:07:46-04:00'},
            {'action': u'registration',
             'actor': None,
             'timestamp': u'2001-01-25T00:00:00-05:00'}],
 'handle': u'NET-66-68-0-0-1',
 'ip_version': u'v4',
 'links': [u'https://rdap.arin.net/registry/ip/066.068.000.000',
           u'https://whois.arin.net/rest/net/NET-66-68-0-0-1'],
 'name': u'RR-SOUTHWEST-2BLK',
 'notices': [{'description': u'By using the ARIN RDAP/Whois service, you are agreeing to the RDAP/Whois Terms of Use',
              'links': [u'https://www.arin.net/whois_tou.html'],
              'title': u'Terms of Service'}],
 'parent_handle': u'NET-66-0-0-0-0',
 'raw': None,
 'remarks': [{'description': u'ADDRESSES WITHIN THIS BLOCK ARE NON-PORTABLE',
              'links': None,
              'title': u'Registration Comments'}],
 'start_address': '66.68.0.0',
 'status': None,
 'type': None}
```

**Comment by secynic** [comment]
Your top result is a legacy whois output, and your ipwhois query is for RDAP. There is an uncomfortable, but necessary adjustment needed when moving from legacy whois to RDAP. It should be in your RDAP result under result['objects']['RRSW']['contact']['name'].

I am going to leave this open, so I remember to update the readme, to note the differences in data results for legacy whois vs RDAP.

**Comment by secynic** [comment]
Merged to dev in #228
